.goreleaser.yml

```diff
@@ -17,9 +22,57 @@ builds:
     env:
       - CGO_ENABLED=0
     ldflags:
-      - -s -w -X main.version={{if .IsSnapshot}}dev{{else}}{{.Version}}{{end}} -X main.commit={{.ShortCommit}} -X main.date={{.Date}}
+      - -s -w -X main.version={{if .IsSnapshot}}{{.Env.SNAPSHOT_VERSION}}{{else}}{{.Version}}{{end}} -X main.commit={{.ShortCommit}} -X main.date={{.Date}}
     binary: influx
 
-# Do not make github release
-release:
-  disable: true
+# TODO: Do we want to publish deb/rpms for the CLI before we release 2.1.0 of the server?
+# How do we handle conflicts between the two until then?
+nfpms:
+  - id: influx-cli
+    formats:
+      - deb
+      - rpm
+    bindir: /usr/bin
+    overrides:
+      rpm:
+        replacements:
+          amd64: x86_64
+          arm64: aarch64
+          armhf: armv7hl
+        file_name_template: influx-cli-nightly.{{ .Arch }}
```

Comment:
Self review: for all the name templates here, need to template out the version instead of hard-coding "nightly"